src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -80,7 +79,7 @@
 [assembly: EntryPointModule(typeof(PavCalibratorTrainerFactory))]
 [assembly: EntryPointModule(typeof(PlattCalibratorTrainerFactory))]
 
-namespace Microsoft.ML.Internal.Calibration
+namespace Microsoft.ML.Calibrator
```

Comment:
>Calibrator [](start = 23, length = 10)

If we really want to have another namespace that's fine I guess, but if the namespace name is `Microsoft.ML.Trainers` then this should probably be `Microsoft.ML.Calibrators`, not `Calibrator`.

Reply:
Should we do `Microsoft.ML.StaticPipe"s"`?

Reply:
let's do that as part of: 
[https://github.com/dotnet/machinelearning/issues/2746](https://github.com/dotnet/machinelearning/issues/2746)

---
In reply to: [260485716](https://github.com/dotnet/machinelearning/pull/2724#discussion_r260485716) [](ancestors = 260485716)